### PR TITLE
partial RoughnessMipmapper fix

### PR DESCRIPTION
**Pull request by elalish**
I noticed a WebGL error (`GL ERROR :GL_INVALID_VALUE : glCopyTexSubImage2D: bad dimensions.`) that I don't think was actually affecting the output at all, but was easily fixed when I realized my loop was going one level too deep.

However, I'm still running into a strange problem where generating the roughness mipmaps is somehow affecting my AO channel. I removed the dispose, since that seemed like it could be the problem, but with no luck. Honestly, I'm sick of hunting down weird problems with this code and having it run on all the MV models even though it makes a noticeable difference on  only a small number. 

The final nail for me is that this can't even work on KTX2 textures, and those in fact must include their mipmaps in the file, which means they can be properly generated there to account for the smoothing of the normal map. So I'm now planning to [remove](https://github.com/google/model-viewer/pull/2935) this from MV and tell anyone who wants it to use KTX2 instead and hopefully a tool exists that will generate mipmaps properly rather than blindly. So @mrdoob, if you'd like to remove RoughnessMipmapper from three.js entirely, that's fine with me.

### CODE REVIEW

**Comment by WestLangley** [comment]
I support removing `RoughnessMipmapper`, too.

**Comment by mrdoob** [comment]
I think we should remove it indeed 👍

**Comment by chubei-oppen** [comment]
I know this is outdated but if someone was still using r136 and ran into this GL error, this post may be of some help.

TL;DR: Remove https://github.com/mrdoob/three.js/blob/1a241ef10048770d56e06d6cd6a64c76cc720f95/examples/jsm/utils/RoughnessMipmapper.js#L90

Explaination:

This line will make the `newRoughnessTexture`'s size the same as original `roughnessMap`. However, following code is still assuming its size is `width x height`, which is calcuated by the maximum of original `roughnessMap` and `normalMap`. So if the material's `roughnessMap` and `normalMap` are of different sizes, you're in trouble.
